File: exs/mundo_1/c++/012.cpp
/*

012 - Calculando descontos
  Faça um algoritmo que leia o preço de um produto e mostre seu novo preço, com
  5% de desconto

*/

#include <iostream>

using namespace std;

int main() {
  double preco, desconto;

  cin >> preco;

  desconto = preco * 0.05;

  cout << "O novo preço, com desconto, é de " << preco-desconto << "\n";
  return 0;
}

File: exs/mundo_1/golang/012.go
/*
Desafio 012

Problema: Faça um algoritmo que leia o preço de um produto e
          mostre seu novo preço, com 5% de desconto.

Resolução do problema:
*/

package main

import (
	"fmt"
)

func main(){
	var preco float64
	fmt.Print("Digite o preço do produto: ")
	fmt.Scanln(&preco)

	desconto := preco * 0.05 // 5% = 0.05
	precoComDesconto := preco - desconto
	fmt.Printf("O produto com %s de desconto, fica %.2f reais.", "5%", precoComDesconto)
}


